src/palladio/Utils.cpp

```diff
@@ -69,6 +69,13 @@ std::basic_string<C> callAPI(FUNC f, size_t initialSize) {
 
 } // namespace
 
+std::vector<const wchar_t*> toPtrVec(const std::vector<std::wstring>& wsv) {
+	std::vector<const wchar_t*> pw(wsv.size());
+	for (size_t i = 0; i < wsv.size(); i++)
```

Comment:
classic use case for std::transform, also see https://github.com/Esri/serlio/blob/ab8f5e675dd164285af62f28603664e579a5e1fa/src/serlio/utils/Utilities.h#L87